api/buf.yaml

```diff
@@ -2,7 +2,7 @@ version: v1
 deps:
     - buf.build/googleapis/googleapis:d1a849b8f8304950832335723096e954
     - buf.build/opencensus/opencensus
-    - buf.build/beta/prometheus
+    - buf.build/prometheus/client-model
```

Comment:
i think this wont work - the proto itself needs to be updated

Reply:
Yeah, I am updating it at local, but seems I cannot access https://api.buf.build/.

Reply:
wierd - any ideas why ?

Reply:
I am behind corporate proxy, let me establish it first.

Reply:
wierd that this didnt fail in CI - i will test again - i wasnt able to updated it when i tried before